agent/engine/docker_container_engine.go

```diff
@@ -371,6 +336,50 @@ func (dg *dockerGoClient) pullImage(image string, authData *api.RegistryAuthenti
 	return nil
 }
 
+func (dg *dockerGoClient) filterDebugOutput(pullDebugOut *io.PipeReader, pullBegan chan bool, image string) {
+	// pullBeganOnce ensures we only indicate it began once (since our channel will only be read 0 or 1 times)
+	pullBeganOnce := sync.Once{}
+
+	reader := bufio.NewReader(pullDebugOut)
+	var line string
+	var pullErr error
+	var statusDisplayed time.Time
+	for pullErr == nil {
+		line, pullErr = reader.ReadString('\n')
+		if pullErr != nil {
+			break
+		}
+		pullBeganOnce.Do(func() {
+			pullBegan <- true
+		})
+
+		now := time.Now()
+		if !strings.Contains(line, "[=") || now.After(statusDisplayed.Add(pullStatusSuppressDelay)) {
+			// skip most of the progress bar lines, but retain enough for debugging
+			log.Debug("Pulling image", "image", image, "status", line)
```

Comment:
i think we should be able to replace the `log.*` with `seelog.*` in these changes.

Reply:
Fixed.